readthedocs/templates/projects/environmentvariable_detail.html

```diff
@@ -18,9 +18,15 @@
 {% block project_edit_content %}
 
   <p>
-    {% blocktrans trimmed %}
-      The value of the environment variable is not shown here for security purposes.
-    {% endblocktrans %}
+    {% if environmentvariable.public %}
+      {% blocktrans trimmed with value=environmentvariable.value %}
+        Value of the environment variable: {{ value }}
```

Comment:
I meant to show something like "privacy level: public" but maybe showing the value isn't a bad idea p:

Reply:
Yup, that’s what I was thinking. And an overview about which ones are public and which ones are private exists.

Reply:
I think we should probably add a note that the variable would be available in builds from PRs or not